common/presenters/events-to-timeline-component.js

```diff
@@ -0,0 +1,66 @@
+const i18n = require('../../config/i18n')
+const addTriggeredEvents = require('../helpers/events/add-triggered-events')
+const setEventDetails = require('../helpers/events/set-event-details')
+
+const getItem = ({
+  component,
+  labelClasses,
+  heading,
+  description,
+  timestamp,
+  byline,
+}) => {
+  return {
+    component,
+    label: {
+      classes: labelClasses,
+      html: heading,
+    },
+    classes: 'app-timeline__item',
+    html: description,
+    datetime: {
+      timestamp,
+      type: 'datetime',
+    },
+    byline: {
+      html: byline,
+    },
+  }
+}
+
+const eventsToTimelineComponent = (moveEvents, move) => {
+  moveEvents = addTriggeredEvents(moveEvents)
+
+  moveEvents = setEventDetails(moveEvents, move)
+
+  const items = moveEvents.map(event => {
+    const { details, event_type: eventType } = event
+    const statusChange = i18n.exists(`events::${eventType}.statusChange`)
+    const labelClasses = statusChange ? 'moj-badge' : undefined
+
+    const heading = i18n.t(`events::${eventType}.heading`, details)
+    const description = i18n
+      .t(`events::${eventType}.description`, details)
+      .replace(/^<br>/, '')
```

Comment:
Whats the reason behind stripping `br`s here? Are they not something we are manually adding in the translation content anyway?

Reply:
Some strings that get added together are prefixed with a `<br>`. This enables them to be used as the first item or where a previous string doesn't get output and not insert additional space.

Reply:
Added a comment to explain the intent

Reply:
That doesn't feel like a great approach but if it's working with the sets of data we can leave it for now. I might have been better to wrap them in a containing element and use CSS to control spacing.